verilog/analysis/checkers/always_comb_blocking_rule.cc

```diff
@@ -43,17 +42,18 @@ using verible::matcher::Matcher;
 // Register AlwaysCombBlockingRule
 VERILOG_REGISTER_LINT_RULE(AlwaysCombBlockingRule);
 
-absl::string_view AlwaysCombBlockingRule::Name() {
-  return "always-comb-blocking";
-}
-const char AlwaysCombBlockingRule::kTopic[] = "combinational-logic";
-const char AlwaysCombBlockingRule::kMessage[] =
+static const char kMessage[] =
     "Use only blocking assignments in \'always_comb\' combinational blocks.";
 
-std::string AlwaysCombBlockingRule::GetDescription(
-    DescriptionType description_type) {
-  return absl::StrCat("Checks that there are no occurrences of ",
-                      "non-blocking assignment in combinational logic.");
+const LintRuleDescriptor& AlwaysCombBlockingRule::GetDescriptor() {
+  static LintRuleDescriptor d{
```

Comment:
static const?

Reply:
Done. And there were another 4-5 cases where I forgot that. Fixed.